experiment/ml/analyze/http.go

```diff
@@ -35,8 +35,9 @@ import (
 )
 
 func serveOnPort(ctx context.Context, storageClient *storage.Client, predictor *predictionClient, port int, timeout time.Duration) error {
+	mux := http.NewServeMux()
```

Comment:
For the record, it looks like using our own ServeMux avoids using the DefaultServeMux which is used otherwise. References: https://stackoverflow.com/a/36249058/437583 https://pkg.go.dev/net/http#hdr-Servers